### PR TITLE
Update from voxpupuli modulesync_config

### PR DESCRIPTION
**Pull request by b4ldr**
Add initial modsync config

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -7,6 +7,12 @@
   "source": "https://github.com/dalen/puppet-dnsquery.git",
   "project_page": "https://github.com/dalen/puppet-dnsquery",
   "issues_url": "https://github.com/dalen/puppet-dnsquery/issues",
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 6.0.0 < 8.0.0"
```
Comment (bastelfreak) [inline metadata.json:13]: please go to 6.1.0 as minimal version. That version has support for detecting systemd services that require a reload.

File: metadata.json [hunk 2]
```diff
@@ -7,6 +7,12 @@
   "source": "https://github.com/dalen/puppet-dnsquery.git",
```
Comment (bastelfreak) [inline metadata.json:7]: please also adjust this to the new urls

File: metadata.json (hunk 1 above)
Comment (b4ldr) [reply metadata.json]: done

File: metadata.json (hunk 2 above)
Comment (b4ldr) [reply metadata.json]: done

File: metadata.json [hunk 3]
```diff
@@ -1,12 +1,18 @@
 {
-  "name": "dalen-dnsquery",
+  "name": "voxpupuli-dnsquery",
   "version": "3.0.0",
   "author": "Erik Dalen <erik.gustav.dalen@gmail.com>",
```
Comment (bastelfreak) [inline metadata.json:4]:
```suggestion
  "author": "Vox Pupuli",
```

**Comment by bastelfreak** [comment]
@b4ldr thanks for the work. can you please rebase the other PRs so the CI works?

**Comment by b4ldr** [comment]
both rebased now

**Pull request merged**